### DISCUSSION TITLE
Visitor Analytics

### DISCUSSION THREAD

**Discussion by root9191** [JulianPrieber/littlelink-custom#223] [In progress]
Hello,
i just installed the newest Release 3.0.1, but since yesterday the Visitor Analytics are at 0:
![image](https://user-images.githubusercontent.com/47309435/202235795-16e88dec-374d-40f6-a4b6-1472d9f4965f.png)

Is there something wrong in my LLC Instance?

**Comment by JulianPrieber** [comment]
The visitor analytics are bound to your littlelink name (@admin). If you change that, this will reset your analytics.

**Comment by root9191** [comment]
My Littlelink Name is Custom - Can i change it from @admin to my name?

**Comment by JulianPrieber** [comment]
You can change it as much as you want, just keep ind mind that this will reset your analytics data.

**Comment by root9191** [comment]
Where do i do this? In the .env File or advanced-config?

**Comment by JulianPrieber** [comment]
Right here ![chrome_h28pGU1520](https://user-images.githubusercontent.com/60265788/202237751-4f57e632-8ef0-4d2e-86a1-39e7d33ddd7b.png)

**Comment by root9191** [comment]
Yes here stands my custom Name, also before the Update. But the counter stays at 0.

**Comment by JulianPrieber** [comment]
Right. And it will always reset to 0 when you change that name.

Did you change that name in the last couple of days?

**Comment by root9191** [comment]
Nope, just when i installed it first and then never again.
I have also set the Home Url to my Custom Name so the Address looks cleaner. Could this be related to?

**Comment by root9191** [comment]
And i have emojis in my Display Name, if this could be related to!?

**Comment by JulianPrieber** [comment]
Hmm, I'm not sure. 

I didn't implement this system, so this is also new to me. The only thing I know about is the littlelink name thing.
This might be related to changing Home Url, I'd have to do some testing to be certain.

**Comment by root9191** [comment]
Ok is there something i can check on my Side? Maybe there are some write access problems on my Hosting.

**Comment by JulianPrieber** [comment]
I mean, it's definitely none of that. 
It would be interesting if you could reproduce the error and find what's causing it.

**Comment by root9191** [comment]
Ok if there is a logfile or something else i could check?
I also redirected the "Home" Page to my basic Domain.

**Comment by JulianPrieber** [comment]
There is, but I doubt you will find something useful in there

**Comment by root9191** [comment]
OK kinda interesting why this is not working. And you can not reproduce it?

**Comment by JulianPrieber** [comment]
I haven't really tried reproducing it yet, I'm still working on the next version.

**Comment by root9191** [comment]
Okok. So i just made a new Instance on my Webhosting and also a Docker Instance locally. Just the Standard Installation without Restoring or anything but the Visitors still stay at 0. The clicks for the links are working fine.

**Comment by JulianPrieber** [comment]
Maybe the update broke the visitor count?

It might be interesting to see if the visitors get tracked in the database.

**Comment by root9191** [comment]
Could not be because i started from scratch with the newest Version..

**Comment by JulianPrieber** [comment]
I mean, you still are using the latest release, so you kinda have to have the updated installed somehow

**Comment by JulianPrieber** [comment]
![chrome_9ERC68dq5d](https://user-images.githubusercontent.com/60265788/202503476-ffdc82d1-0ab1-4e41-b6ac-59ca2cfff8ed.png)

All versions I have on the latest version have the same issue, so it might actually have been the update.
It cant really have been the update from 3.0.0 to 3.0.1 so I'm not sure at what point this system did break.

**Comment by root9191** [comment]
Ok this Visitor Analytics is since Version 3 or earlier?

**Comment by JulianPrieber** [comment]
Yeah, since version 3, but it had multiple iterations in the beta stage.

**Comment by escuco** [comment]
Same issue here, the visitor analytics (today, week, ...) remaining at "0". The counter "total links" and "top links" are counting fine.

But for me it's not problem, since I'm using Matomo.
